sdk/python/src/datastar_py/sse.py

```diff
@@ -139,25 +139,25 @@ def execute_script(
         auto_remove: bool = True,
         attributes: Optional[list[str]] = None,
         event_id: Optional[int] = None,
-        retry_duration: int = consts.DefaultSseRetryDuration,
+        retry_duration: int = consts.DEFAULT_SSE_RETRY_DURATION,
     ):
         data_lines = []
-        data_lines.append(f"data: {consts.AutoRemoveDatalineLiteral} {auto_remove}")
+        data_lines.append(f"data: {consts.AUTO_REMOVE_DATALINE_LITERAL} {auto_remove}")
 
         if attributes:
             data_lines.extend(
-                f"data: {consts.AttributesDatalineLiteral} {attribute}"
+                f"data: {consts.ATTRIBUTES_DATALINE_LITERAL} {attribute}"
                 for attribute in attributes
-                if attribute.strip() != consts.DefaultExecuteScriptAttributes
+                if attribute.strip() != consts.DEFAULT_EXECUTE_SCRIPT_ATTRIBUTES
             )
 
         data_lines.extend(
-            f"data: {consts.ScriptDatalineLiteral} {script_line}"
+            f"data: {consts.SCRIPT_DATALINE_LITERAL} {script_line}"
             for script_line in script.splitlines()
         )
 
         return ServerSentEventGenerator._send(
-            consts.EventType.EventTypeExecuteScript,
+            consts.EventType.EXECUTE_SCRIPT,
```

Comment:
Please revert to the convention used by all SDKs, in this case it should be `EVENT_TYPE_EXECUTE_SCRIPT`.

Reply:
Ignore me.